src/SeerUK/Composer/PimcoreInstaller.php

```diff
@@ -122,6 +122,36 @@ public static function installWebsite(Event $event)
     }
 
     /**
+     * copy the .htaccess file if it does not exist
+     *
+     * @param Event $event
+     *
+     * @return void
+     */
+    public static function copyHtAccessFile(Event $event)
+    {
+        list($installPath, $vendorPath) = self::prepareBaseDirectories($event);
+
+        if (file_exists($vendorPath . "/pimcore/pimcore/.htaccess")) {
+            $from = $vendorPath . "/pimcore/pimcore/.htaccess";
+        } elseif (file_exists($vendorPath . "/pimcore/pimcore/website_example/.htaccess")) {
+            $from = $vendorPath . "/pimcore/pimcore/website_example/.htaccess";
+        } else {
+            // we don't know about the .htaccess file.
+            return;
+        }
+
+        $to = $installPath . "/.htaccess";
```

Comment:
This check should probably be above the rest of this method's body, because it's the most likely to return early, and help prevent unnecessary stuff happening.